app.js

```diff
@@ -1,8 +1,27 @@
-function hello(name) {
-  const greeting = `hello ${name}`;
-  return greeting;
+const kaholo = require("kaholo-plugin-library");
+const gcpCli = require("./gcpcli");
+
+async function runCommand(parameters) {
+  let additionalFlags = "--format=json ";
+  if (parameters.zone) {
+    additionalFlags += `--zone=${parameters.zone} `;
+  }
+  if (parameters.project) {
+    additionalFlags += `--project=${parameters.project} `;
+  }
+
+  try {
+    const result = await gcpCli.execute(
+      parameters.credentials,
+      parameters.command,
+      additionalFlags,
+    );
+    return result.stdout;
+  } catch (error) {
+    return Promise.reject(error.stderr ?? error);
```

Comment:
If stderr is a string (which it probably is) the plugin's method will throw that type, while it's an `Error` we'd like to catch outside.